packages/client/hmi-client/src/components/workflow/tera-dataset-node.vue

```diff
@@ -76,4 +72,34 @@ section {
 	flex-direction: column;
 	max-width: 400px;
 }
+
+span {
+	font-size: var(--font-caption);
+	color: var(--text-color-subdued);
+}
+
+:deep(.p-datatable > .p-datatable-wrapper) {
+	font-size: xx-small;
+}
+
+:deep(.p-datatable .p-datatable-thead > tr > th) {
+	font-size: xx-small;
+}
+
+:deep(.p-datatable.p-datatable-sm .p-datatable-tbody > tr > td) {
+	padding: 0 5px 0 5px;
+	background-color: var(--gray-50);
+}
+
+:deep(.p-datatable.p-datatable-sm .p-datatable-thead > tr > th) {
+	background-color: var(--gray-50);
+}
+
+:deep(.p-datatable .p-datatable-tbody > tr > td) {
+	border: none;
+}
+
+:deep(.p-datatable .p-datatable-thead > tr > th) {
+	border: none;
+}
```

Comment:
Why not updating the scss for p-datatable and add an extra classname that you can just apply, like `datatable-small` or something better?

Reply:
Good idea